packages/testing/src/test/chai-retry-plugin.unit.ts

```diff
@@ -217,6 +217,30 @@ describe('chai-retry-plugin', () => {
             await expect(resultFunction).retry().to.not.change(myObj, 'comas');
             expect(getCallCount()).to.equal(2);
         });
+
+        it('.respondTo()', async () => {
+            class EchoService {
+                echo: (() => void) | undefined;
+            }
+
+            setTimeout(() => {
+                EchoService.prototype.echo = () => {
+                    return;
+                };
+            }, 900);
+
+            await expect(() => new EchoService(), 'should respond')
+                .retry({ timeout: 1000 })
+                .to.respondTo('echo');
+        });
+
+        it('.keys()', async () => {
```

Comment:
what exactly is being tested here?

Reply:
@yurii-ve added this test to cover more `chai`'s built-in asserters, `.keys()` in this case

You are right, `testArray` sounds better) Also removed redundant empty objects